docs/user/publication/index.md

```diff
@@ -0,0 +1,91 @@
+
+Once you complete all the fields, you should no longer see any warnings. Your content is then ready to be published.
+
+:::note
+You can publish your content even if there are still some warnings, but we encourage you to try to complete all the fields (also in all the elements of your content, not just in the root folder) to maximise visibility in the Graasp Library, and to increase the chances that other people will use your content.
+:::
+
+### Publish Your Content
+
+When you are ready, click the `Publish` button to start publishing your collection. It is important to understand that your folder and all its content will be visible to anyone at the end of this process. If your folder is private, the following modal will be displayed, asking you to change its visibility to public.
+
+![Change the item's visibility](make_it_public.jpeg)
+
+Once you click on the `Publish` button, the collection will first go through an approval process. This may take a few moments depending on the number of elements. You'll see a "Pending" status while your content is being validated.
+
+:::info
+If you don't see an update soon, try to refresh the page.
+:::
+
+![The publication is pending](publication_pending.jpeg)
+
+:::note
+We want to make sure that our library is a safe and enjoyable place for everyone. That's why we use AI to check all published content, including yours. We try to respect your privacy as much as possible, using local models. This means that we do not share your private data with any third party. If this should change, we will inform you.
+:::
+
+Once your content is approved, it will be published in the Graasp Library. You'll see a confirmation message when this happens.
+
+:::info
+If the status stays "Pending" for a while, just refresh the page.
+:::
+
+![The publication was successful](publication_succeed.jpeg)
+
+We want to ensure the Graasp Library is a safe and appropriate space for everyone. If your content contains violence or sexually explicit material, it won't be published. You're welcome to revise and resubmit content that aligns with our guidelines. Don't hesitate to contact us at admin@graasp.org if you believe your content is legitimate.
+
+![The publication has failed](publication_failed.jpeg)
+
+### Unpublish Your Content
+
+If for any reason, you would like to remove your content from the Graasp Library, click on the `Unpublish` button to achieve this action. Your content will then no longer be available in the Graasp Library.
+
+:::warning
+You will still need to change the visibility to `private` in the item's settings if you want to restrict access. Unpublishing your content will only remove it from the library; its visibility will remain public.
+:::
+
+### Viewing Your Content in the Library
+
+The easiest way to view your published content is by clicking the `View on Graasp Library` button. This will redirect you directly to your content in the Graasp Library.
```

Comment:
```suggestion
The easiest way to view your published content is by clicking the `View on Graasp Library` button. This will redirect you directly to your content in Graasp Library.
```